packages/wrangler/src/pages/dev.tsx

```diff
@@ -245,6 +274,15 @@ export const Handler = async ({
 
 	await scriptReadyPromise;
 
+	if (scriptPath === "") {
```

Comment:
Pretty sure the "without" case covers it well enough that this will never get hit, but fair enough: we can add this if you think there's a risk.

Reply:
You're probably right, but it felt like a good sanity check. I originally added this because of TS complaining about `scriptPath` being used before it was assigned after changing the `if()` above it. Ended up switching it to default to an empty string, but this check still felt like a good idea.